adm_program/modules/dates/dates.php

```diff
@@ -133,6 +132,13 @@
             window.open("'.ADMIDIO_URL.FOLDER_MODULES.'/dates/dates.php?view_mode=print&view=' . $getView . '&mode=' . $getMode . '&headline=' . $getHeadline . '&cat_id=' . $getCatId . '&id=' . $getId . '&date_from=' . $dates->getParameter('dateStartFormatEnglish') . '&date_to=' . $dates->getParameter('dateEndFormatEnglish') . '", "_blank");
         });', true);
 
+    $page->addJavascript('
+        function submitParticipationForm(datId, mode){
```

Comment:
Please change to: `function submitParticipationForm(datId, mode) {` (whitespace in front of `{`)